src/Bedrock.Framework.Experimental/Protocols/Tls/Hashs/Hash.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Tls.Hashs
+{
+    internal abstract class Hash
```

Comment:
What's this?

Reply:
That is the same as below the provider below will provide an instance of this (from say OpenSsl or BCrypt)